src/store/modules/wallet.js

```diff
@@ -344,6 +355,7 @@ const actions = {
           e.timestamp = new Date(e.timestamp);
           return e;
         })
+        .filter(e => e.action === '/cosmos.nft.v1beta1.MsgSend')
```

Comment:
What are the other events that exists other than `/cosmos.nft.v1beta1.MsgSend`? we filtered `action_type` in API already